ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/layout/TipsLayout.kt

```diff
@@ -14,28 +14,27 @@ import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.style.TextAlign
-import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import com.airbnb.lottie.compose.LottieAnimation
 import com.airbnb.lottie.compose.LottieCompositionSpec
 import com.airbnb.lottie.compose.LottieConstants
 import com.airbnb.lottie.compose.animateLottieCompositionAsState
 import com.airbnb.lottie.compose.rememberLottieComposition
 import org.cru.godtools.base.ui.theme.GodToolsTheme
-import org.cru.godtools.tutorial.R
 
-@Preview(showBackground = true)
 @Composable
-internal fun TipsTutorialLayout() = GodToolsTheme() {
+internal fun TipsTutorialLayout(anim: Int, title: Int, body: Int, body2: Int = 0) = GodToolsTheme() {
+
     Column(
         modifier = Modifier.fillMaxSize(),
         verticalArrangement = Arrangement.Center,
         horizontalAlignment = Alignment.CenterHorizontally
     ) {
 
         val composition by rememberLottieComposition(
+
             LottieCompositionSpec
-                .RawRes(org.cru.godtools.tutorial.R.raw.anim_tutorial_tips_people)
+                .RawRes(anim)
         )
```

Comment:
now that `anim` is being passed in as a parameter, there isn't a lot of code here, so it should be able to be condensed to a single line